en/08_Changelogs/rc/5.3.0-rc1.md

```diff
@@ -101,7 +101,7 @@ A full list of module versions included in CMS Recipe 5.3.0-rc1 is provided belo
 
 </details>
 
-## Release Candidate
+## Release candidate
```

Comment:
A few linting fixes in this file

Reply:
If you rebase this should be done already - I won't hold up merging for that though.

Reply:
Have rebased